cvrf2csaf/section_handlers/acknowlegments.py

```diff
@@ -0,0 +1,34 @@
+
+from ..common.common import SectionHandler
+
+
+class Acknowledgments(SectionHandler):
+    def __init__(self):
+        super().__init__()
+
+    def _process_mandatory_elements(self, root_element):
+
+        if hasattr(root_element, 'Acknowledgment'):
+            # at least one entry in list of Acknowledgments
+            pass # No field is mandatory due to CVRF.xsd; version 1.2
+
+
+    def _process_optional_elements(self, root_element):
+        # optional values
+        names = []
+        organization = []
+        summary = []
+        urls = []
+        for ack in root_element.Acknowledgment:
+            names.append(ack.Name.text)
+            if hasattr(ack, 'Organization'):
+                organization.append(ack.Organization.text)
+            if hasattr(ack, 'Summary'):
+                summary.append(ack.Summary.text)
+            if hasattr(ack, 'Url'):
+                urls.append(ack.Url.text)
+
+        self.csaf['names'] = names
```

Comment:
@sustefil: Correct. This applies to all optional attributes: If they are empty, they should not be present.

Reply:
yes - As written [here](https://github.com/csaf-tools/CVRF-CSAF-Converter/pull/34#discussion_r785732177), this can easily be done before returning the complete `self.csaf` and safes us quiet some repeatable lines. 

But sure, can be implemented in each element as well.